spec/requests/api/v1/activities_spec.rb

```diff
@@ -1,7 +1,69 @@
 require 'rails_helper'
 
-RSpec.describe "Api::V1::Activities", type: :request do
-  describe "GET /index" do
-    pending "add some examples (or delete) #{__FILE__}"
+RSpec.describe 'Api::V1::Activities', type: :request do
+  let(:role) { FactoryBot.create(:role, name: 'admin', description: 'admin user') }
```

Comment:
Si el factory de `role` ya existe solo necesitarias:
```suggestion
  let(:role) { create(:role, name: 'admin', description: 'admin user') }
```
La mismo deberías revisar en los otros casos.